app/Http/Controllers/Bible/BibleFileSetsController.php

```diff
@@ -178,22 +179,31 @@ public function showChapter(
             $verse_start,
             $verse_end
         ];
-
+        // fixes data issue where text filesets use the same filesetID
+        if (in_array($type, ['text_plain', 'text_format'])) {
```

Comment:
could you add two more text types to this? text_html, text_usx. Also please consolidate the logic so the code is not duplicated (eg line 183 and 199)